src/meta/NebulaSchemaProvider.cpp

```diff
@@ -89,6 +89,15 @@ void NebulaSchemaProvider::addField(folly::StringPiece name,
                             static_cast<int64_t>(fields_.size() - 1));
 }
 
+void NebulaSchemaProvider::addField(folly::StringPiece name, nebula::cpp2::ValueType&& type,
+    nebula::cpp2::Value defaultValue) {
```

Comment:
why is not defaultValue  right value,  but type is right value?

Reply:
Mistake, thanks.